### ISSUE TITLE
feature request, fixed screen orientation

### ISSUE THREAD

**Issue by treakiandroid** [TobiasBielefeld/Simple-Solitaire#1]
Hi and thanks for this new grate game in fdroid!,

it reminsds me realy much at kPatience (same end animation and card art). I like the handling very much. but there is one thing i am missing: i have seen that the app woekes geate in portrait and in landscape mode, but normaly my device is fixed to portrait mode so that i have to switch to landscape orientation if i want to enyoj the app best. So i would be very happy if you could add an orientation setting where you can select defaulr (as of the android system suggests), fixed landscape, fixed portrait ans landscape upside down mode. thanks in advance. and also i am happy to see more Patience's (like frecell, spider and yukon) in the future.

thanks and keep up the good work

**Comment by TobiasBielefeld** [comment]
Hello, I am really happy to see that you like my app :)

I added the fixed orientation setting and made a new release. I uploaded it to the Google Play Store, but I don't know how long it takes to be updated in the F-Droid Store. Because they check for a new release on their own. So it could take some time until the update reaches F-Droid.

I also want to add some more games in my app, but I currently don't have enough time for that. So it's on my to do list :D
